### PR TITLE
Feat: update docs

### PR DESCRIPTION
**Pull request by BrendaHijmans**
update docs and resources (templates) to fit with 3.x data model

### CODE REVIEW

File: docs/catalogue/cat_cohort-data-manager.md [hunk 1]
```diff
@@ -4,16 +4,15 @@
...
-cohorts and in the data banks composing the data sources (aka 'source variables'); and mappings to common data models 
-(aka 'target variables'). Its purpose is to facilitate pooled data analysis of multiple cohorts together 
-[Fortier et al, 2017](https://pubmed.ncbi.nlm.nih.gov/27272186/), and multi-data
+cohorts and data sources; and mappings to common data models. Its purpose is to facilitate pooled data analysis of multiple cohorts
+[Fortier et al, 2017](https://pubmed.ncbi.nlm.nih.gov/27272186/) and multi-data
 source studies [Gini et al, 2020](https://pubmed.ncbi.nlm.nih.gov/32243569/).
 
 - The metadata of cohorts include descriptive information such as contact details, name of the cohort, and high-level
   summary of contents and cohort design. The metadata of data sources, of the corresponding data banks and of the
   institutions that provide access to them, include descriptive information such as contact details, reason for
```
Comment (erikzwart) [inline docs/catalogue/cat_cohort-data-manager.md:13]: do we need to rename institutions to organisations (datamodel 3)?

File: docs/catalogue/cat_cohort-data-manager.md [hunk 2]
```diff
@@ -4,16 +4,15 @@
...
   existence of the data banks, the prompt for the records in the data bank, and lag time for updating and accessing data.
-- The metadata of the source variables (also known as 'data dictionary') can be considered as a codebook or data
+- The metadata of the source variables can be considered the codebook or data
   dictionary of a cohort (e.g. ALSPAC) and of the tables which make up a data source's data bank(s) (e.g. the Danish
   Healthcare Registries).
 - Similarly, the common data models (or 'target variables') can be considered the codebook for a network of institutions
```
Comment (erikzwart) [inline docs/catalogue/cat_cohort-data-manager.md:18]: link to Data harmonasition: DataSHIELD (https://www.datashield.org/)

File: docs/catalogue/cat_cohort-data-manager.md [hunk 3]
```diff
@@ -35,232 +34,188 @@ metadata are entered in the live catalogue.
...
+Open your staging area, navigate to 'Tables' and open the table 'Cohorts'. Your cohort id and name are already 
 filled out. Click on the pencil sign next to this entry to start editing your cohort rich metadata by filling out 
-the form.
+the form. 'Subcohorts' and 'Collection events' should also be filled out through this route. You can fill them out 
+in subsections inside the 'Cohorts' form.
 
 ### Define metadata of cohorts or data sources
 
 This section explains how to submit the 'source variables' + 'mappings from source variables to target variables' into
 the Data Catalogue. Expected users of this 'how to' are data managers within the institutions with access to cohorts or
```
Comment (erikzwart) [inline docs/catalogue/cat_cohort-data-manager.md:62]: rename institutions to organisation?

File: docs/catalogue/cat_cohort-data-manager.md [hunk 4]
```diff
@@ -35,232 +34,188 @@ metadata are entered in the live catalogue.
...
 data sources. This document assumes you have received login details for upload of your metadata. You can also watch
-this [*instruction video*](https://www.youtube.com/watch?v=b_Ef_Uiw1gE&amp;ab_channel=MOLGENIS).
+this [*instruction video*](https://www.youtube.com/watch?v=b_Ef_Uiw1gE&amp;ab_channel=MOLGENIS). Note that this video used the old dictionary model.
```
Comment (erikzwart) [inline docs/catalogue/cat_cohort-data-manager.md:64]: model version 2.x

File: docs/catalogue/cat_cohort-data-manager.md [hunk 5]
```diff
@@ -35,232 +34,188 @@ metadata are entered in the live catalogue.
...
 upload. You can download this 
 [*filled out example*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/SourceDictionary_testCohort.xlsx) 
-as a reference for filling out the template.
-
-<sup>Note that there is no sheet for *AllSourceVariables*. This table is a generic listing of all variables entered for
-the cohort; it shows *SourceVariables* and *RepeatedSourceVariables* in one table. </sup>
+as a reference for filling out the template. Note that there is no sheet for *AllVariables*. This table is a generic listing of all variables entered for
+the cohort; it shows *Variables* and *RepeatedVariables* in one table.
 
 It is good practice to try adding a few variables to the template first and see whether your upload succeeds. To
```
Comment (erikzwart) [inline docs/catalogue/cat_cohort-data-manager.md:79]: might need to add additional troubleshooting page if uploading of variables fails

File: docs/catalogue/cat_cohort-data-manager.md [hunk 6]
```diff
@@ -35,232 +34,188 @@ metadata are entered in the live catalogue.
...
-| description | Table description |
-| unitOfObservation | Defines what each record in this table describes |
-| numberOfRows | Count of the number of records in this table |
+| resource \* | Cohort or data source that this table belongs to |
+| name \* | Unique dataset or table name |
+| label | Dataset label |
+| description | Dataset description |
+| unit of observation | Defines what each record in this dataset describes |
+| number of rows | Count of the number of records in this dataset |
+| keywords<sup>1</sup> | Enables grouping of variables into topics and helps to display variables in a tree | Find list to choose from in Catalogue |
```
Comment (erikzwart) [inline docs/catalogue/cat_cohort-data-manager.md:101]: link to CatalogueOntologue keyword list/options

File: docs/catalogue/cat_cohort-data-manager.md [hunk 7]
```diff
@@ -35,232 +34,188 @@ metadata are entered in the live catalogue.
...
 
 #### Define harmonisations
 
-We use the [*Mappings* template*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/SourceMappings.xlsx) to
+We use the [*Mappings* template*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/Mappings.xlsx) to
 describe the harmonisations. The 
-[*Mappings* template*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/SourceMappings.xlsx) consists of two
-sheets (TableMappings and VariableMappings). It is used to define the mappings from source variables to target
+[*Mappings* template*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/Mappings.xlsx) consists of two
+sheets (DatasetMappings and VariableMappings). It is used to define the mappings from source variables to target
```
Comment (erikzwart) [inline docs/catalogue/cat_cohort-data-manager.md:177]: need space between Dataset mappings and Variable mappings

File: docs/catalogue/cat_cohort-data-manager.md [hunk 8]
```diff
@@ -35,232 +34,188 @@ metadata are entered in the live catalogue.
...
-| match | Whether the harmonisation is partial, complete or NA (non-existent) | Find list to choose from in CatalogueOntologies (StatusDetails) |
+| source \* | Databank or cohort id |  |
+| source dataset \* | Source table name | Datasets must be predefined in the *Datasets* sheet in the SourceDictionary template |
+| source variables | Source variable name(s) | Variables must be predefined in the _Variables_ sheet in the SourceDictionary template; When multiple variables are mapped together use a comma-separated list, e.g. v1,v2,v3 |
+| source variables other datasets.dataset | Other source tables | When using variables from multiple other datasets, use a comma-separated list, e.g. dataset1,dataset2,dataset3<sup>1</sup> |
+| source variables other datasets.name | Source variable(s) from other datasets than filled out under source dataset | When using variables from multiple other datasets, use a comma-separated list, the order corresponding to the order of the datasets they are derived from specified under source variables from other datasets.dataset<sup>1</sup> |
+| target \* | Name of the target common data model  | e.g. LifeCycle_CDM, LongITools_CDM |
```
Comment (erikzwart) [inline docs/catalogue/cat_cohort-data-manager.md:210]: is it clear for users where to find the CDM options?

File: docs/catalogue/cat_network-data-manager.md [hunk 9]
```diff
@@ -47,123 +44,98 @@ section [Upload metadata](cat_network-data-manager.md#upload-metadata) to the Da
...
+The network's datasets are defined in the *Datasets* sheet. Columns with an asterisk (\*) after their name are mandatory.
 
 | *Column name* | *Description* | *Remarks* |
 | --- | --- | --- |
-| dataDictionary.resource \* | Name of the model. | e.g LifeCycle_CDM, LongITools_CDM or ATHLETE_CDM |
-| dataDictionary.version \* | Target data dictionary version this table is part of. | e.g. 1.0.1 |
-| name \* | Unique table name | |
-| label | Table label | |
-| description | Table description | |
+| resource \* | Id of the <b>model</b>. | e.g LifeCycle_CDM, LongITools_CDM or ATHLETE_CDM |
```
Comment (erikzwart) [inline docs/catalogue/cat_network-data-manager.md:66]: columns: unit of observation, keywords and number of rows are not explained

File: docs/catalogue/cat_network-data-manager.md [hunk 10]
```diff
@@ -47,123 +44,98 @@ section [Upload metadata](cat_network-data-manager.md#upload-metadata) to the Da
...
-| dataDictionary.resource \* | Name of the model that contains this variable | e.g LifeCycle_CDM, LongITools_CDM or ATHLETE_CDM |
-| dataDictionary.version \* | Target data dictionary version this variable is part of | e.g. 1.0.1 |
-| table \* | Table that contains the variable | Tables must be predefined in the TargetTables sheet |
-| name \* | Variable name, unique within a table | |
+| resource \* | Id of the <b>model</b> that contains this variable | e.g LifeCycle_CDM, LongITools_CDM or ATHLETE_CDM |
+| dataset \* | Dataset that contains the variable | Datasets must be predefined in the _Datasets_ sheet |
+| name \* | Variable name, unique within a dataset | |
 | label | Human readable variable label | |
 | format | The data type of the variable | Find list to choose from in CatalogueOntologies |
 | unit<sup>1</sup> | Unit in case of a continuous or integer format | Find list to choose from in CatalogueOntologies |
```
Comment (erikzwart) [inline docs/catalogue/cat_network-data-manager.md:84]: missing columns: references, permitted value, order, repeats, notes,

File: docs/catalogue/cat_network-data-manager.md [hunk 11]
```diff
@@ -47,123 +44,98 @@ section [Upload metadata](cat_network-data-manager.md#upload-metadata) to the Da
...
+are defined in the *Repeated variables* sheet. Defining your repeated variables using this sheet
 will limit the amount of information that has to be repeated when filling out repeated variables. This sheet is
 optional.
 
 | *Column name* | *Description* | *Remarks* |
 | --- | --- | --- |
-| dataDictionary.resource \* | Name of the model that contains this variable | e.g LifeCycle_CDM or ATHLETE_CDM |
-| dataDictionary.version \* | Target data dictionary version this variable is part of | e.g. 1.0.1 |
-| table \* | Table name | e.g. core |
+| resource \* | Id of the <b>model</b> that contains this variable | e.g LifeCycle_CDM or ATHLETE_CDM |
```
Comment (erikzwart) [inline docs/catalogue/cat_network-data-manager.md:128]: missing column: label

File: docs/catalogue/cat_network-data-manager.md [hunk 12]
```diff
@@ -47,123 +44,98 @@ section [Upload metadata](cat_network-data-manager.md#upload-metadata) to the Da
...
-The sheet *Subcohorts* is optional. Here you may describe populations that can be linked to collection events.
-
-| *Column name* | *Description* | *Remarks* |
-| --- | --- | --- |
-| resource | Name of the model that contains this subcohort | e.g LifeCycle_CDM |
-| name \* | Name of the subpopulation or subcohort | e.g. mothers or children |
-| description | Subpopulation description | |
-
-<sup>Table 7. Description of the columns that can be filled out for Subpopulations. * = mandatory</sup>
+| is repeat of.table \* | Dataset that contains the variable that is repeated | Datasets must be predefined in the _Datasets_ sheet; e.g. core |
```
Comment (erikzwart) [inline docs/catalogue/cat_network-data-manager.md:131]: not .table but .dataset

File: docs/catalogue/cat_network-data-manager.md [hunk 13]
```diff
@@ -47,123 +44,98 @@ section [Upload metadata](cat_network-data-manager.md#upload-metadata) to the Da
...
+| is repeat of.name \* | Name of the variable that is repeated | Variables must be predefined in the _Variables_ sheet; e.g. height\_0 |
+| collection event.resource | Refer to the network that contains the collection event | e.g. LifeCycle |
+| collection event.name | Refer to the name of a collection event | The collection event needs to be predefined via online forms, as described above; e.g. y1 or y2 |
+| since version | Version of the data model when this variable was introduced | e.g. 1.0.0 or 2.1 |
```
Comment (erikzwart) [inline docs/catalogue/cat_network-data-manager.md:135]: since and until not defined in TargetDictionary.xlsx (Repeated variables)

**Review by erikzwart — CHANGES_REQUESTED** [review]
see notes on cat_cohort-data-manager.md and cat_network-data-manager.md

**Comment by erikzwart** [comment]
[pr update docs](https://trello.com/c/gDhKjm5o/616-pr-update-docs)

File: docs/catalogue/cat_cohort-data-manager.md (hunk 1 above)
Comment (BrendaHijmans) [reply docs/catalogue/cat_cohort-data-manager.md:13]: Organisations is a bit more broad then Institutions, you can fill out departments, companies, research institutes etc.

File: docs/catalogue/cat_cohort-data-manager.md (hunk 3 above)
Comment (BrendaHijmans) [reply docs/catalogue/cat_cohort-data-manager.md:62]: i think deleting 'within the institutions' from the sentence will be better

File: docs/catalogue/cat_cohort-data-manager.md (hunk 5 above)
Comment (BrendaHijmans) [reply docs/catalogue/cat_cohort-data-manager.md:79]: out of scope for this PR and story

File: docs/catalogue/cat_cohort-data-manager.md [hunk 14]
```diff
@@ -35,232 +34,188 @@ metadata are entered in the live catalogue.
 
 You will need credentials to log in and upload metadata. 
 
-LifeCycle, ATHLETE and LongITools use [*data-catalogue-staging*](https://data-catalogue-staging.molgeniscloud.org).  
+Cohorts in projects such as ATHLETE, IPEC and LongITools use [*data-catalogue-staging*](https://data-catalogue-staging.molgeniscloud.org).
```
Comment (erikzwart) [inline docs/catalogue/cat_cohort-data-manager.md:37]: data-catalogue-staging not in use anymore

File: docs/catalogue/cat_cohort-data-manager.md [hunk 15]
```diff
@@ -35,232 +34,188 @@ metadata are entered in the live catalogue.
...
 When you log in, you will be able to see at least the following databases:
 
-- <b> DataCatalogue</b>: The catalogue data, in which you can search for target variables to map to.
+- <b>DataCatalogue</b>: The catalogue data, in which you can search for target variables to map to.
 - <b>CatalogueOntologies</b>: This database contains the look-up list that you need for filling out some columns in the
   templates, e.g. format or unit. If you need to add anything to these look-up lists, contact us
   at [molgenis-support](mailto:molgenis-support@umcg.nl).
-- <b>SharedStaging</b>: A communal staging area in which e.g. Contacts and Institutions are added and edited.
-- <b>Your own database</b>: use this to upload the templates once you have filled them out.
+- <b>Your own database </b>: (here: testCohort) Use this to fill out cohort rich metadata and to upload the templates once you have filled them out.
```
Comment (erikzwart) [inline docs/catalogue/cat_cohort-data-manager.md:47]: You also have access to SharedStaging?

File: docs/catalogue/cat_cohort-data-manager.md
Comment (erikzwart) [inline docs/catalogue/cat_cohort-data-manager.md:1]: url broken on line 233

File: docs/catalogue/cat_cohort-data-manager.md [hunk 16]
```diff
@@ -35,232 +34,188 @@ metadata are entered in the live catalogue.
...
 | description | Description of the harmonisation | |
 | syntax | Syntax used for this harmonisation | |
 
-<sup>*Table 9. Description of the columns that can be filled out for VariableMappings. * = mandatory <sup>1</sup>see sheet VariableMappings in the 
-[*example template*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/SourceMappings_testCohort.xlsx)
+<sup>*Table 6. Description of the columns that can be filled out for Variable mappings. * = mandatory; 1 = see sheet Variable mappings in the 
+[*example template*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/Mappings_testCohort.xlsx)
 for an example on how to fill this out (last line) </sup> 
 
 ### Upload metadata
```
[the remark this answers is not in the document]
Comment (BrendaHijmans) [reply docs/catalogue/cat_cohort-data-manager.md:221]: this is a different PR I think, should make a story for this

File: docs/catalogue/cat_cohort-data-manager.md (hunk 8 above)
Comment (BrendaHijmans) [reply docs/catalogue/cat_cohort-data-manager.md:210]: added links

File: docs/catalogue/cat_network-data-manager.md (hunk 10 above)
Comment (BrendaHijmans) [reply docs/catalogue/cat_network-data-manager.md]: repeats are not to be filled out here but are rendered automatically in the catalogue. the other columns are not explained, bt also not necessary to fill out. May explain them later on.

File: docs/catalogue/cat_network-data-manager.md (hunk 13 above)
Comment (BrendaHijmans) [reply docs/catalogue/cat_network-data-manager.md:141]: check

**Review by erikzwart — APPROVED** [review]
Url of the staging need to be fixed but users will be redirected anyway.
All links to the templates work and are correct.
SharedStaging will be used in this data model version right?

**Pull request merged**